### PR TITLE
added grid component

### PR DESCRIPTION
**Pull request by sk-ip**
Please review, this is the grid component

### CODE REVIEW

File: component/grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  grid-template-columns: repeat(auto-fit, minmax(150px, 1fr));
+}
+
+.grid-3 {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(150px, 1fr));
+}
+
+.grid-item {
+  border: 1px solid black;
```
Comment (Aditi0505) [inline component/grid/grid.css:28]: Use hex value for consistency.

**Review by Aditi0505 — APPROVED** [review]
Left one suggestion. Rest looks good to me.

File: component/grid/grid.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  background-color: #17a2b8;
```
Comment (ravikumar0403) [inline component/grid/grid.css:29]: use variables for colors

**Review by kotesh-arya — APPROVED** [review]
Using CSS Variables for colors will be very helpful and saves you from many issues caused by in-consistency
, rest is good enough for a merge 👍🏼

**Review by ravikumar0403 — APPROVED** [review]

**Review by shivamSspirit — APPROVED** [review]
good look to me

File: component/grid/grid.css (hunk 1 above)
Comment (sk-ip) [reply component/grid/grid.css:28]: ok

File: component/grid/grid.css (hunk 2 above)
Comment (sk-ip) [reply component/grid/grid.css:29]: thanks

**Pull request merged**